src/stories/components/delegate-summary/delegate-summary.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Please, do not use global ignores as it could be forgotten, use inline ignores instead

Reply:
done